src/btree/bt_read.c

```diff
@@ -400,8 +400,14 @@ __wt_page_in_func(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags
                     evict_skip = true;
                 else if (ret == EBUSY) {
                     WT_NOT_READ(ret, 0);
-                    WT_STAT_CONN_INCR(session, page_forcible_evict_blocked);
-                    stalled = true;
+                    /* Do not retry to evict the pages during reconciliation. */
+                    if (F_ISSET(session, WT_SESSION_NO_RECONCILE)) {
+                        WT_STAT_CONN_INCR(session, cache_eviction_force_no_retry);
+                        evict_skip = true;
+                    } else {
+                        WT_STAT_CONN_INCR(session, page_forcible_evict_blocked);
+                        stalled = true;
+                    }
                     break;
```

Comment:
Due to the break below, the read operation sleeps at least once. Instead of a break, how about using `continue` similar to successful eviction to improve performance?

Reply:
@kommiharibabu Yes, continuing will not let to sleep even once. Fixed in 7b3f328ac7ea398f94fe8caaeb33f9562a3c9daf